core/src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/HttpGoogleServicesDAO.scala

```diff
@@ -142,7 +142,7 @@ class HttpGoogleServicesDAO(val clientSecrets: GoogleClientSecrets,
   override def updateBucketIam(bucketName: GcsBucketName,
                                policyGroupsByAccessLevel: Map[WorkspaceAccessLevel, WorkbenchEmail],
                                userProject: Option[GoogleProjectId],
-                               iamPolicyVersion: Int = 1
+                               iamPolicyVersion: Int = 3
```

Comment:
I was able to lock workspaces with `iamPolicyVersion` set to 1, but when unlocking I got an error that the policy version is 3 and I can't use a lower version. The bucket IAM policy version in google said it was 1 though, so I'm not sure what is going on 😞 

It seems like if we always use 3 we won't run into issues because it will use version 1 when necessary: https://cloud.google.com/iam/docs/allow-policies#valid

```If the allow policy does not contain any conditions, then IAM always returns a version 1 allow policy, regardless of the version number in the request.```

Reply:
A bucket with fast pass grants will have conditional policies (which use version 3), so maybe that's why I saw the error